src/KeyVault/KeyVault/Commands/Certificate/Policy/SetAzureKeyVaultCertificatePolicy.cs

```diff
@@ -271,6 +272,19 @@ public override void ExecuteCmdlet()
         {
             if (ShouldProcess(Name, Properties.Resources.SetCertificatePolicy))
             {
+
+                // Manually Validate `RenewAtNumberOfDaysBeforeExpiry` and `RenewAtPercentageLifetime`
+                if (RenewAtNumberOfDaysBeforeExpiry.HasValue && (RenewAtNumberOfDaysBeforeExpiry < 1 || RenewAtNumberOfDaysBeforeExpiry > int.MaxValue))
+                {
+                    throw new ArgumentOutOfRangeException(nameof(RenewAtNumberOfDaysBeforeExpiry), "Value must be between 1 and int.MaxValue.");
```

Comment:
throw AzPS Exception, see https://github.com/Azure/azure-powershell-common/tree/main/src/Common/Exceptions

Reply:
Updated this! Thanks for pointing this out!